src/actions/index.test.js

```diff
@@ -0,0 +1,39 @@
+import * as actions from './index'
+
+describe('actions', () => {
+  it('should return a type of IS_LOADING, with isLoading', () => {
+    const isLoading = true
+    const expected = {
+      type: 'IS_LOADING',
+      isLoading
+    }
+
+    const result = actions.isLoading(isLoading)
+
+    expect(result).toEqual(expected)
+  })
+
+  it('should return a type of HAS_ERRORED, with an id', () => {
+    const hasErrored = false
+    const expected = {
+      type: 'HAS_ERRORED',
+      hasErrored
+    }
+
+    const result = actions.hasErrored(hasErrored)
+
+    expect(result).toEqual(expected)
+  })
+
+  it('should return a type of FETCH_DATA_SUCCESS, with a filter', () => {
```

Comment:
Here the it block states you are returning a filter, when it is returning movies. The test looks great!

Reply:
good catch, thanks! Fixed!